docs/dapps/sdk/transaction.md

```diff
@@ -0,0 +1,131 @@
+---
+sidebar_position: 10
+title: Transaction API Guide
+sidebar_label: Transaction API Guide
+---
+
+## Unsigned Transaction Structure
+
+An unsigned transaction contains the following elements:
+
+* `version`: This field specifies the version of the transaction format being used. This allows the network to handle different formats as the protocol evolves
+* `networkId`: This field identifies the specific network the transaction is intended for. This ensures that the transaction is processed on the correct network
+* `scriptOpt`: This optional field contains the script code to be executed as part of the transaction. This is often used for transactions involving smart contracts
+* `gasAmount`: This field specifies the amount of gas that the transaction can consume
+* `gasPrice`: This field indicates the price per unit of gas that the sender is willing to pay. This determines the total transaction fee and can affect the transaction's priority in the network
+* `inputs`: This field lists the sources of funds for the transaction. Each input typically references a previous transaction output
+* `fixedOutputs`: This field defines the recipients and amounts of the transaction. Each output specifies an address and the amount of funds being sent to that address
+
+## Signed Transaction Structure
+
+Unsigned transactions can only be submitted to the Alephium network after being signed. A signed transaction contains the following elements:
+
+* `unsigned`: The unsigned transaction
+* `scriptExecutionOk`: This field indicates whether the script transaction executed successfully
+* `contractInputs`: This field contains all the contract assets required for executing the transaction
+* `generatedOutputs`: This field contains all outputs generated by the contract
+* `inputSignatures`: This field contains the signatures for the inputs used in the transaction
+* `scriptSignatures`: This field contains the signatures that are used by the asset script
+
+## Transaction Builder
+
+In Alephium, you can first build unsigned transactions, then sign the unsigned transactions, and finally submit the signed transactions to the network. The `TransactionBuilder` is used to build unsigned transactions.
+
+### Building Transfer Transactions
+
+```typescript
+import { TransactionBuilder } from '@alephium/web3'
+
+const nodeUrl = 'http://127.0.0.1:22973'
+const builder = TransactionBuilder.from(nodeUrl)
+const senderAddress = '1DrDyTr9RpRsQnDnXo2YRiPzPW4ooHX5LLoqXrqfMrpQH'
+const publicKey = '0381818e63bd9e35a5489b52a430accefc608fd60aa2c7c0d1b393b5239aedf6b0'
```

Comment:
[Nit]
Maybe its easier for the reader if we use `signer.buildTransferTx`, since they would most likely need to figure out how to get the corresponding public key for the address if they follow the example. WDYT?

Reply:
Or perhaps we can add a short description of how to get the public key